README.md

```diff
@@ -163,7 +163,63 @@ To run the specs:
 2. Run `npx jasmine-browser-runner`.
 3. Visit <http://localhost:8888>.
 
-## Saucelabs support
+## Remote Grid support (Saucelabs, BrowserStack, etc.)
+
+jasmine-browser-runner can run your Jasmine specs on a remote grid
+provider like [Saucelabs](https://saucelabs.com/), 
+[BrowserStack](https://browserstack.com) or your own Selenium Grid.
+To use a remote grid hub, set the `browser` object
+in your config file as follows:
+
+```json
+// jasmine-browser.json
+{
+  // ...
+  // BrowserStack
+  "browser": {
+    "url": "https://hub-cloud.browserstack.com/wd/hub",
+    "browserName": "safari",
+    "bstack:options": {
+      "browserVersion": "13",
+      "os": "OS X",
+      "osVersion": "Snow Leopard",
+      "local": "true",
+      "localIdentifier": "tunnel ID",
+      "debug": "true",
+      "userName": "your BrowserStack username",
+      "accessKey": "your BrowserStack access key"
+    }
+  }
+}
+```
+```json
+// jasmine-browser.json
+{
+  // ...
+  // Saucelabs
```

Comment:
I think it would be better to have something like this:

```
 "browser": {
    "name": "safari",
    "useRemoteSeleniumGrid": true,
    "remoteSeleniumGrid": {
       "url": "https://ondemand.saucelabs.com/wd/hub",
       "platformName": "macOS 12",
       "sauce:options": {
         "tunnel-identifier": "tunnel ID",
         "userName": "your Saucelabs username",
         "accessKey": "your Saucelabs access key"
      }
    }
  }
```

Advantages:
* Browser is always specified via the "name" property, not "name" in some configurations and "browserName" in others
* Can tell at a glace which properties are specific to remote Selenium grid and which are more general
* It's more obvious which config property determines whether to use a remote Selenium grid
* The separate "useRemoteSeleniumGrid" boolean makes it easier to have a single config file template that's sometimes used for local runs and sometimes remote. Jasmine-core does this, to allow the same command to run tests locally and on Saucelabs in CI.

That might make config processing a little more complicated, but I think the added clarity is worth it.